package.json

```diff
@@ -10,8 +10,6 @@
     "benchmark": "cross-env DEBUG=Eleventy:Benchmark* npx @11ty/eleventy",
     "build": "rm -rf _site && npm run generate:tts-languages && npx @11ty/eleventy",
     "build-preview": "rm -rf _site && npm run generate:tts-languages && PREVIEW_BUILD=1 npx @11ty/eleventy",
-    "deploy": "export $(grep -v -E '^\\s*#|^\\s*$' .env | sed 's/\\s*#.*//' | xargs) && node deploy.js",
-    "build-deploy": "npm run build && npm run deploy",
```

Comment:
<https://github.com/NabuCasa/support#how-to-contribute> .env reference here, and the example file should be removed as well.